src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -216,6 +540,27 @@ public void externalBinaryContentHandlingAttribute(final String uri) {
                         "process the request. At least one of the following handling attributes must be supported: " +
                         "copy, redirect, and/or proxy.",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final List<String> acceptedHandlingTypes = getAcceptedHandlingTypes(uri);
+
+        final List<String> acceptedAndExpected = acceptedHandlingTypes.stream()
+                .filter(HANDLINGS::contains)
+                .collect(Collectors.toList());
+
+        assertThat("Accept-External-Content-Handling header contains at least one expected handling",
+                acceptedAndExpected.size(), greaterThanOrEqualTo(1));
+
+        // Test that at least one of the handlings specified by the serve is supported
+        for (final String handling : acceptedAndExpected) {
+            final Headers headers = new Headers(
+                    new Header(LINK, externalContentHeader(externalUri, handling)),
+                    new Header(SLUG, info.getId()));
```

Comment:
Please avoid using the `Slug` header where possible as support for it is not required AFAIK.